index.html

```diff
@@ -3,10 +3,32 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" type="text/css" href="index.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>veilarbdetaljerfs</title>
+    <style>
+        html,
+        body {
+          display: flex;
+          flex-direction: column;
+          height: 100%;
+          margin: 0;
+        }
+
+        #main {
+          display: flex;
+          flex-direction: column;
+          height: 100%;
+        }
+
+        #veilarbpersonflatefs-root {
+          width: 100%;
+          min-height: 174px;
+        }
+		</style>
   </head>
   <body>
+    <img src="public/internflate20211122.png" id="veilarbpersonflatefs-root" />
     <div id="root"></div>
```

Comment:
Dette blir litt HTML/UU teori. I HTML så har man mange forskjellige elementer og mange av disse er såkallede "semantiske"-elementer (aka. meningsbærende elementer). `div` er ikke semantisk siden det basically bare betyr "en boks". Et semantisk element som kan gi mer mening å bruke her i stedet for `div` er `main`. `main` betyr "en boks" men i tillegg betyr den "den mest relevante boksen" (sagt på en annen måte: "i denne boksen finnes hovedinnholdet på denne siden"). 

Man bør som hovedregel bruke semantiske elementer når man kan og det gir mening (og heller bruke `div` dersom det ikke finnes alternativer). Det er flere fordeler med semantiske elementer; for det første er det lettere for andre som leser koden å skjønne hva som er hva. I tillegg kan assisterende teknologi (AT) som skjermlesere osv. gjøre seg nytte av semantiske elementer, da AT-ene vil kunne tilby brukeren mer kontekstuell informasjon.

```suggestion
    <main id="root"></main>
```

Reply:
Om man vil lese mer om `main`-elementet/semantiske elementer/UU så er dette noen gode kilder:

* https://aksel.nav.no/god-praksis/artikler/utvikling#h5c24c8fe671f
* https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/main_role
* https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#good_semantics